.mergify.yml

```
@@ -13,6 +13,20 @@ pull_request_rules:
             git merge upstream/{{base}}
             git push upstream {{head}}
             ```
+  - name: backport patches to 8.16 branch
+    conditions:
+      - merged
+      - base=main
+      - label=v8.16.0
+    actions:
+      backport:
+        assignees:
+          - "{{ author }}"
+        branches:
+          - "8.x"
```

Since `8.x` is now `8.17`, shouldn't this be `8.16`?

```suggestion
          - "8.16"
```

---

Actually, scratch this...the current value (`8.x`) might be correct since we haven't cut an `8.17` branch from `8.x` yet. I  saw https://github.com/elastic/security-docs/pull/5958 and automatically jumped to the thought that the `8.x` branch is now synonymous with `8.17`.

---

I think so too, though it's maybe moot since this PR is against the 8.16 branch so it won't be backporting to itself :)

---

Yeah, that's a good point, @lcawl. I would still advise that we change `8.x` to `8.16` though, so it's clear that 8.16 is the target branch for this action.